src/providers/weights.rs

```diff
@@ -72,7 +72,7 @@ fn calculate_chain_weight(
     let Availability(chain_success, chain_failure) = chain_availability;
```

Comment:
not sure but does this need to be updated, too?

```suggestion
    let Availability(chain_success, chain_failure * chain_failure) = chain_availability;
```

Reply:
Doing this would mean, that if chain Y fails, chain X will be punished exponentially. 
We can consider it, but wouldn't do it for now

Reply:
If the whole provider is down (rate limit) it should react quickly enough, but if only one chain fails (like Pokt - mainnet eth recently) we don't want the whole provider to go down too quickly